src/overlays/actors/ovl_En_Ik/z_en_ik.c

```diff
@@ -930,9 +928,12 @@ void EnIk_Update(Actor* thisx, GlobalContext* globalCtx2) {
             func_800B9010(&this->actor, NA_SE_EV_ICE_FREEZE - SFX_FLAG);
         }
     }
-    EnIk_ArmorDestroyed(this, globalCtx);
+    EnIk_UpdateArmor(this, globalCtx);
 }
 
+static s8 D_8092C1A8[] = { -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, -1, 0,  -1, -1, -1,
```

Comment:
I wouldn't make this static unless it needs to go in a function or have a generic name, purely for mapfile reasons.